templates/templates.go

```diff
@@ -95,46 +90,12 @@ func (t *Templater) RenderTemplate(w io.Writer, data interface{}, mainTmpl Templ
 	return t1.Execute(w, data)
 }
 
-func (t *Templater) RenderEmail(emailTemplate Template) *template.Template {
-	return template.Must(template.New(emailTemplate.String()).ParseFS(tmpls, emailTemplate.String()))
+func (t *Templater) RenderEmail(emailTemplate Template) (*template.Template, error) {
```

Comment:
NITPICK: Does this actually render the email, or does it just create a templater for it?

Reply:
Renamed